ydb/docs/en/core/yql/reference/yql-core/syntax/_includes/create_table.md

```diff
@@ -113,6 +113,17 @@ CREATE TABLE my_table (
 ```
 {% endif %}
 
+{% if feature_temp_tables %}
+{% if feature_olap_tables %}#{%endif%}## Creating a temporary table {#row-additional}
+```sql
+CREATE TEMPORARY TABLE table_name (
```

Comment:
Will adding indexes to such table with `ALTER` also make them temporary? If that's the case, then the alter article needs updating too.